pkgs/development/python-modules/mmcv/default.nix

```diff
@@ -102,12 +81,23 @@ buildPythonPackage rec {
   nativeBuildInputs = [
     ninja
     which
-  ] ++ lib.optionals cudaSupport [ cuda-native-redist ];
+  ];
 
-  buildInputs = [
-    pybind11
-    torch
-  ] ++ lib.optionals cudaSupport [ cuda-redist ];
+  buildInputs =
+    [
+      pybind11
+      torch
+    ]
+    ++ lib.optionals cudaSupport (
+      with cudaPackages;
+      [
+        cuda_cudart # cuda_runtime.h
+        cuda_cccl # <thrust/*>
+        libcublas # cublas_v2.h
+        libcusolver # cusolverDn.h
+        libcusparse # cusparse.h
+      ]
```

Comment:
This is probably equivalent to `(getOutput "cxxdev" torch)`

Reply:
i tried that but then i get 

```
<command-line>: fatal error: cuda_runtime.h: No such file or directory
```